### PR TITLE
FIO Address registration changes/fixes. Fio domain registration link.

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

File: src/modules/UI/components/ControlPanel/Component/Main.js [hunk 1]
```diff
@@ -306,15 +304,12 @@ const LogoutButton = ({ onPress }) => {
   )
 }
 
-const goToFioAddressesScene = async (fioAddresses: FioAddress[] | null) => {
-  if (!fioAddresses || !fioAddresses.length) return Actions[Constants.FIO_ADDRESS_REGISTER]()
-  return Actions[Constants.FIO_ADDRESS_LIST]()
-}
-const FioAddressButton = ({ fioAddresses }) => {
+const goToFioAddressesScene = () => Actions[Constants.FIO_ADDRESS_LIST]()
```
Comment (paullinator) [inline src/modules/UI/components/ControlPanel/Component/Main.js]: Unnecessary function goToFioAddressesScene. Just pass `Actions[Constants.FIO_ADDRESS_LIST]` to the onPress prop

File: src/components/scenes/FioAddressListScene.js [hunk 2]
```diff
@@ -54,8 +55,8 @@ export class FioAddressListScene extends Component<Props, State> {
     refreshAllFioAddresses(() => this.checkForFioAddresses())
```
Comment (paullinator) [inline src/components/scenes/FioAddressListScene.js:55]: refreshAllFioAddresses does not need a callback. You already have redux state fioAddressesLoading and you can simply look at this to determine whether the addresses have been loaded and if not show a spinner

Comment (andreyvEze) [reply src/components/scenes/FioAddressListScene.js]: i'm using cb to navigate to the address register scene. Changed using componentDidUpdate

File: src/modules/UI/components/ControlPanel/Component/Main.js (hunk 1 above)
Comment (andreyvEze) [reply src/modules/UI/components/ControlPanel/Component/Main.js]: done

File: src/components/scenes/FioAddressRegisterScene.js [hunk 3]
```diff
@@ -38,13 +37,12 @@ export type State = {
...
   fioWallets: EdgeCurrencyWallet[],
   fioPlugin: EdgeCurrencyConfig,
-  defaultFiatCode: string,
   isConnected: boolean
 }
 
 export type DispatchProps = {
   changeFioAddressName: (fioAddressName: string) => void,
-  createCurrencyWallet: (walletName: string, walletType: string, fiatCurrencyCode: string) => any
+  createFioWallet: () => Promise<any>
```
[the remark this answers is not in the document]
Comment (andreyvEze) [reply src/components/scenes/FioAddressRegisterScene.js]: done

**Pull request merged**